UI/Panels/Input/DigInputMuxPanel.cs

```diff
@@ -0,0 +1,284 @@
+                        (panel as MobiFlight.UI.Panels.Action.PmdgEventIdInputPanel).syncFromConfig(_config.onPress as PmdgEventIdInputAction);
+                    else if (!isOnPress && _config != null && _config.onRelease != null)
+                        (panel as MobiFlight.UI.Panels.Action.PmdgEventIdInputPanel).syncFromConfig(_config.onRelease as PmdgEventIdInputAction);
+
+                    break;
+
+                case InputConfig.JeehellInputAction.Label:
+                    panel = new MobiFlight.UI.Panels.Action.JeehellInputPanel();
+                    if (isOnPress && _config != null && _config.onPress != null)
+                        (panel as MobiFlight.UI.Panels.Action.JeehellInputPanel).syncFromConfig(_config.onPress as JeehellInputAction);
+                    else if (!isOnPress && _config != null && _config.onRelease != null)
+                        (panel as MobiFlight.UI.Panels.Action.JeehellInputPanel).syncFromConfig(_config.onRelease as JeehellInputAction);
+
+                    break;
+
+                case InputConfig.VJoyInputAction.Label:
+                    panel = new MobiFlight.UI.Panels.Action.VJoyInputPanel();
+                    if (isOnPress && _config != null && _config.onPress != null)
+                        (panel as MobiFlight.UI.Panels.Action.VJoyInputPanel).syncFromConfig(_config.onPress as VJoyInputAction);
+                    else if (!isOnPress && _config != null && _config.onRelease != null)
+                        (panel as MobiFlight.UI.Panels.Action.VJoyInputPanel).syncFromConfig(_config.onRelease as VJoyInputAction);
+                    break;
+
+                case MobiFlight.InputConfig.LuaMacroInputAction.Label:
+                    panel = new MobiFlight.UI.Panels.Action.LuaMacroInputPanel();
+                    if (isOnPress && _config != null && _config.onPress != null)
+                        (panel as MobiFlight.UI.Panels.Action.LuaMacroInputPanel).syncFromConfig(_config.onPress as LuaMacroInputAction);
+                    else if (!isOnPress && _config != null && _config.onRelease != null)
+                        (panel as MobiFlight.UI.Panels.Action.LuaMacroInputPanel).syncFromConfig(_config.onRelease as LuaMacroInputAction);
+
+                    break;
+
+                case MobiFlight.InputConfig.RetriggerInputAction.Label:
+                    panel = new MobiFlight.UI.Panels.Action.RetriggerInputPanel();
+                    if (isOnPress && _config != null && _config.onPress != null)
+                        (panel as MobiFlight.UI.Panels.Action.RetriggerInputPanel).syncFromConfig(_config.onPress as RetriggerInputAction);
+                    else if (!isOnPress && _config != null && _config.onRelease != null)
+                        (panel as MobiFlight.UI.Panels.Action.RetriggerInputPanel).syncFromConfig(_config.onRelease as RetriggerInputAction);
+
+                    break;
+
+                case MobiFlight.InputConfig.MSFS2020EventIdInputAction.Label:
```

Comment:
We shall not forget to Merge MSFS2020 Events and Custom Input since they were merged in the mean time